src/assets/scss/Component/_help.scss

```diff
@@ -50,24 +47,18 @@
         .postbox {
           display: none;
 
-          h3 {
-            margin-bottom: 0;
-
-            .hndle {
-              padding-bottom: 12px;
-
-              .spinner {
+          .hndle {
+            padding: 12px;
```

Comment:
Why the move from `padding-bottom` to `padding: 12px`?

Reply:
This is for the size of the title back which contain the text 'Gravity PDF Documentation'. It looks very small.

Reply:
reverted